app/models/index.js

```diff
@@ -1,4 +1,5 @@
 require('./influencer');
+require('./signup');
```

Comment:
Ugh, I still left out a bunch of files, but should be in the latest commit. Sorry about that